common/flavorization/00_ctp_titles_02.txt

```diff
@@ -3261,6 +3261,647 @@ count_tribal_female_hungarian = {
+	top_liege = no
+}
+title_prince_male_ethiopian_empire = {
+	type = character
+	gender = male
+	special = ruler_child
+	tier = empire
+	priority = 130
+	cultures = { ethiopian }
+	governments = { feudal_government clan_government tribal_government }
+	top_liege = no
+}
+title_prince_female_ethiopian_empire = {
+	type = character
+	gender = female
+	special = ruler_child
+	tier = empire
+	priority = 130
+	cultures = { ethiopian }
+	governments = { feudal_government clan_government tribal_government }
+	top_liege = no
+}
+baron_male_republic_ethiopian = {
+	type = character
+	gender = male
+	special = holder
+	tier = barony
+	priority = 103
+	cultures = { ethiopian }
+	governments = { republic_government }
+	top_liege = no
+	only_holder = yes
+}
+barony_republic_ethiopian = {
+	type = title
+	tier = barony
+	priority = 103
+	cultures = { ethiopian }
+	governments = { republic_government }
+	top_liege = no
+}
+
+###############################################
+## Nubian ##
+###############################################
+
+baron_male_nubian = {
+	type = character
+	gender = male
+	special = holder
+	tier = barony
+	priority = 103
+	cultures = { nubian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+baron_female_nubian = {
+	type = character
+	gender = female
+	special = holder
+	tier = barony
+	priority = 103
+	cultures = { nubian }
+	governments = { feudal_government clan_government tribal_government }
+	top_liege = no
+}
+baron_female_holder_nubian = {
+	type = character
+	gender = female
+	special = holder
+	tier = barony
+	priority = 104
+	cultures = { nubian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+	only_holder = yes
+}
+barony_nubian = {
+	type = title
+	tier = barony
+	priority = 103
+	cultures = { nubian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+count_feudalclan_male_nubian = {
+	type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = 113
+	cultures = { nubian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+count_female_nubian = {
+	type = character
+	gender = female
+	special = holder
+	tier = county
+	priority = 113
+	cultures = { nubian }
+	governments = { feudal_government clan_government tribal_government }
+	top_liege = no
+}
+count_feudalclan_female_holder_nubian = {
+	type = character
+	gender = female
+	special = holder
+	tier = county
+	priority = 114
+	cultures = { nubian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+	only_holder = yes
+}
+county_feudalclan_nubian = {
+	type = title
+	tier = county
+	priority = 113
+	cultures = { nubian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+duke_feudalclan_male_nubian = {
+	type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 123
+	cultures = { nubian }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+duke_female_nubian = {
```

Comment:
This should not have tribal? And also add in feudal clan

Reply:
This is the generic term for lady

Reply:
The ruler terms are gender neutral.